db/sql/sql_test.go

```diff
@@ -109,8 +109,8 @@ var _ = Describe("Sql", func() {
 		})
 
 		insertTwoRecords := func() {
-			Expect(tx.Exec(ctx, "INSERT INTO `tests` (`id`, `tenant_id`) values ('id1', 'tenant1')")).To(Succeed())
-			Expect(tx.Exec(ctx, "INSERT INTO `tests` (`id`, `tenant_id`) values ('id2', 'tenant2')")).To(Succeed())
+			Expect(tx.Exec(ctx, "INSERT INTO `tests` (`id`, `tenant_id`, `domain_id`) values ('id1', 'tenant1', 'domain')")).To(Succeed())
```

Comment:
With `test_schemas.yaml` fix, `domain_id` is now required property. Therefore, we need to add domain_id here. Without this, sqlite3 test failed.